src/pytest_split/algorithms.py

```diff
@@ -0,0 +1,106 @@
+from typing import TYPE_CHECKING
+import heapq
+from collections import namedtuple
+
+if TYPE_CHECKING:
+    from typing import List, Dict, Tuple
+    from _pytest import nodes
+
+
+ALGORITHMS = ["duration_based_chunks", "least_duration"]
```

Comment:
I think enum could make more sense here

Reply:
Changed to enum. Please let me know if I made the change you intended.